ruby/lib/trailer_vote/media_types/configuration.rb

```diff
@@ -15,6 +15,36 @@ class Configuration < BaseText
 
       validations do
 
+        version 5 do
+          attribute :configuration do
+            link :self
+            link :place
+            link :push_manifest
+            link :products
+            link :product_lookup
+            link :persona do
+              attribute :href, Types.makeFormattedUrl(:uuid)
+              attribute :templated, TrueClass
+            end
+            link :lookup do
+              attribute :href, Types.makeFormattedUrl(:query)
```

Comment:
Can you use an `identifier` and `authority` parameter instead of one `query` parameter?